server/base/src/main/java/org/apache/accumulo/server/fs/DefaultTieredStorageStrategy.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.server.fs;
+
+import org.apache.accumulo.server.conf.ServerConfigurationFactory;
+import org.apache.accumulo.server.conf.TableConfiguration;
+
+public class DefaultTieredStorageStrategy implements TieredStorageStrategy {
+  @Override
+  public String getVolumeList(VolumeChooserEnvironment env, ServerConfigurationFactory confFactory,
```

Comment:
Agreed. I will change, but do you have a suggestion? The property names the <code>PreferredVolumeChooser</code> uses are "volumes". I could do <code>getVolumes</code> but I feel like the plurality also signifies it should be a collection of some sort

Reply:
Yeah, you're right. When I made that comment, I didn't quite understand yet that it was essentially following the same pattern as the chooser. But now, I think it doesn't matter, because I think this can be refactored without this interface.